test/plots/barPlotTests.ts

```diff
@@ -1 +1,783 @@
 ///<reference path="../testReference.ts" />
+
+describe("Plots", () => {
+  describe("Bar Plot", () => {
+    describe ("orientations", () => {
```

Comment:
I prefer either using an adjective or using the endpoint name for the describe message.  Note that if a test were to fail, we would get this message...

"Plots Bar Plot orientations ..."

Ex. "Plots Bar Plot orientations rejects invalid orientations"

Lots of broken English here.  To respond to this I think the first two pieces can be remedied using our module structure.  However, the third piece just breaks the flow completely.  An endpoint here or an adjective would continue the sentence flow pretty seamlessly into the the `it`